hedera-node/hedera-app/src/main/java/com/hedera/node/app/blocks/impl/BlockStreamBuilder.java

```diff
@@ -812,6 +801,11 @@ public BlockStreamBuilder parentConsensus(@NonNull final Instant parentConsensus
         return this;
     }
 
+    @Override
+    public StreamBuilder triggeringParentConsensus(@NonNull final Instant parentConsensus) {
+        return parentConsensus(parentConsensus);
```

Comment:
nit: I find this line confusing as the same token is used to represent an `Instant` and a method that returns a `BlockStreamBuilder`.  No real reason to change, though

Reply:
Hmm yes will rename method argument in next PR.